### ISSUE TITLE
i18n ?

### ISSUE THREAD

**Issue by morlandi** [Axelrod-Python/DjAxelrod#19]
any interest in setting up internationalization of UI ?

**Comment by meatballs** [comment]
Yes. I think we should.

**Comment by morlandi** [comment]
I can cope with it. 
Just wonder if there are some specific requirements due to the fact that source code has been generated by django-builder.
Probably not: as far as I understand, it was just a one-step operation, and after that we completely forget about django-builder, but I'ld like to hear the opinion of @mmcardle

**Comment by mmcardle** [comment]
Yup, django-builder is just a one step operation to give some boilerplate code, from that point on its all up to you.

I would like to add i18n to django-builder, but that's another topic for another time.

**Comment by morlandi** [comment]
OK, so I'm going to mark it as "in progress", and I'll be back with a PR hopefully in a week or so.

@mmcardle : are you willing to accept contribution in this area for django-builder ?

**Comment by mmcardle** [comment]
Yes i am, i am in the process of getting the source onto github, currently the github project has just the static html that is built for the github.io pages.
